tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -431,10 +440,94 @@ def test_common_workload(self):
                                            driver, (workload, validator),
                                            num_workers=self.CLUSTER_NODES - 1,
                                            topology="ensemble")
-        benchmark.start()
-        benchmark_time_min = benchmark.benchmark_time() + 5
-        benchmark.wait(timeout_sec=benchmark_time_min * 60)
-        benchmark.check_succeed()
+        # Latency spikes detection and retry
+        max_retries = 2
+        try_count = 0
+
+        while try_count < max_retries:
+            try_count += 1
+            self.logger.info(
+                f"Starting benchmark attempt {try_count}/{max_retries}.")
+            # Run the benchmark test
+
+            benchmark.start()
+            benchmark_time_min = benchmark.benchmark_time() + 5
+            benchmark.wait(timeout_sec=benchmark_time_min * 60)
+
+            #benchmark.check_succeed()
+            # check if omb gave errors, but don't process metrics
+            #benchmark.check_succeed(validate_metrics=True, )
+            is_valid, validation_results = benchmark.check_succeed(
+                raise_exceptions=False)
+
+            if is_valid:
+                self.logger.info(
+                    "\n\nBenchmark passed without significant latency issues.\n"
```

Comment:
Don't put newlines in the log like this.

Reply:
Fixed. This was used while testing.